app/src/main/res/layout/tracking_protection_category.xml

```diff
@@ -27,7 +27,7 @@
         android:id="@+id/trackingProtectionCategoryTitle"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/library_item_icon_margin_horizontal"
+        android:layout_marginStart="32dp"
```

Comment:
Could it be better to have a generic dimen for all settings?

Reply:
This will be happening in a follow up PR, see https://github.com/mozilla-mobile/fenix/pull/12087#discussion_r447069606

Reply:
Awesome, thanks :)